pkg/minikube/cruntime/cri.go

```diff
@@ -49,6 +51,24 @@ type crictlImages struct {
 	} `json:"images"`
 }
 
+func CrictlVersion(c CommandRunner) (*semver.Version, error) {
+	rr, err := c.RunCmd(exec.Command("sudo", "crictl", "--version"))
+	if err != nil {
+		return nil, err
+	}
+	stdout := rr.Stdout.String()
+	reg := regexp.MustCompile(`crictl\s*version\s*(v\d*\.\d*.\d*)`)
```

Comment:
```suggestion
	reg := regexp.MustCompile(`crictl\s*version\s*(v\d+\.\d+.\d+)`)
```
Want at least one digit per major, minor, patch